From f82e34083a58b4cb82e93008b486e867c92dcefc Mon Sep 17 00:00:00 2001
From: Romain Rigaux <romain@cloudera.com>
Date: Fri, 1 May 2015 17:15:48 -0700
Subject: [PATCH 1169/1173] HUE-2723 [hive] Listing table information in non
 default DB fails

Conflicts:

	apps/beeswax/src/beeswax/server/hive_server2_lib.py
---
 .../beeswax/src/beeswax/server/hive_server2_lib.py |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/apps/beeswax/src/beeswax/server/hive_server2_lib.py b/apps/beeswax/src/beeswax/server/hive_server2_lib.py
index f08f80a..b5af816 100644
--- a/apps/beeswax/src/beeswax/server/hive_server2_lib.py
+++ b/apps/beeswax/src/beeswax/server/hive_server2_lib.py
@@ -628,9 +628,9 @@ class HiveServerClient:
 
     if self.query_server['server_name'] == 'impala':
       # Impala does not supported extended
-      query = 'DESCRIBE %s' % table_name
+      query = 'DESCRIBE %s.%s' % (database, table_name)
     else:
-      query = 'DESCRIBE EXTENDED %s' % table_name
+      query = 'DESCRIBE EXTENDED %s.%s' % (database, table_name)
     (desc_results, desc_schema), operation_handle = self.execute_statement(query, max_rows=5000, orientation=TFetchOrientation.FETCH_NEXT)
     self.close_operation(operation_handle)
 
-- 
1.7.9.5

